### PR TITLE
Sync for Windows

### LINKED ISSUE #1559: Sync for Windows

### PR DESCRIPTION
**Pull request by fealebenpae**
Closes #1559

### CODE REVIEW

File: Realm/Realm.Sync/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -23,4 +23,5 @@
 [assembly: InternalsVisibleTo("Tests.iOS")]
 [assembly: InternalsVisibleTo("Tests.Android")]
 [assembly: InternalsVisibleTo("Tests.NetCore")]
-[assembly: InternalsVisibleTo("Tests.XamarinMac")]
\ No newline at end of file
+[assembly: InternalsVisibleTo("Tests.XamarinMac")]
+[assembly: InternalsVisibleTo("Tests.UWP")]
```
Comment (nirinchev) [inline Realm/Realm.Sync/Properties/AssemblyInfo.cs]: Should we have `InternalsVisibleTo("Tests.Win32")`?

File: Tests/Tests.UWP/Tests.UWP.csproj [hunk 2]
```diff
@@ -130,6 +130,18 @@
       <Link>realm-wrappers.dll</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="$(SolutionDir)wrappers\build\WindowsStore\$(Configuration)-$(RealmWrappersPlatform)\zlib1.dll" Condition="'$(RealmNoSync)' != 'true'">
```
Comment (nirinchev) [inline Tests/Tests.UWP/Tests.UWP.csproj]: Do we need to make these changes to the Tests.Win32 project as well?

Comment (fealebenpae) [reply Tests/Tests.UWP/Tests.UWP.csproj]: Nope, actually this is no longer required and I should delete it.

Comment (nirinchev) [reply Tests/Tests.UWP/Tests.UWP.csproj]: What about the references to Realm.Sync and Tests.Sync.Shared?

Comment (fealebenpae) [reply Tests/Tests.UWP/Tests.UWP.csproj]: Those are fine, I believe. Do you see sometging wrong?

Comment (nirinchev) [reply Tests/Tests.UWP/Tests.UWP.csproj]: I meant - shouldn't we add them to the Win32 project? Seems like only the UWP one has been updated in this PR.

Comment (fealebenpae) [reply Tests/Tests.UWP/Tests.UWP.csproj]: Huh, I swear I added them, but maybe I forgot to push. I’ll check.

File: NuGet/Realm/Realm.targets [hunk 3]
```diff
@@ -3,4 +3,30 @@
   <PropertyGroup>
     <_RealmNugetNativePath>$(MSBuildThisFileDirectory)..\native\</_RealmNugetNativePath>
   </PropertyGroup>
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
```
Comment (nirinchev) [inline NuGet/Realm/Realm.targets]: Is this still needed?

**Comment by Bo-Yee-Woods** [comment]
await User.LoginAsync(...) never return a user. Whether sync realm is ready for UWP?

**Comment by nirinchev** [comment]
It is, can you open a new issue and fill in the template so we can adequately help.

**Pull request merged**